docs/F-Refinements/portability.md

```diff
@@ -1,94 +1,99 @@
 ---
+id: portability
+title: Portability
 sidebar_position: 7
+slug: /refinements/portability
+description: About Portability
 ---
+
 # Portability
 
 ## Learning Outcomes
 
 After reading this section, you will be able to:
 
-* Explain procedural programming using nontechnical terminology to inform a business person
+- Explain procedural programming using nontechnical terminology to inform a business person
 
 ## Introduction
 
-Mature languages attract international standards.  These standards promote uniformity across host platforms.  Committees of compiler writers, academics, applications developers and representatives from various national organizations review, discuss, negotiate and reach consensus on which elements should be included in the standard definition.  Elements that are excluded are called extensions.  Compiler writers include their *extensions* in their own implementations of the agreed standard. 
+Mature languages attract international standards. These standards promote uniformity across host platforms. Committees of compiler writers, academics, applications developers and representatives from various national organizations review, discuss, negotiate and reach consensus on which elements should be included in the standard definition. Elements that are excluded are called extensions. Compiler writers include their _extensions_ in their own implementations of the agreed standard.
```

Comment:
I believe "applications developers" has to be corrected to "application developers" (the "application" becomes singular).